survey_contact_generation/models/survey_user_input.py

```diff
@@ -72,6 +72,11 @@ def _create_contact_post_process(self, partner):
             subtype_id=self.env.ref("mail.mt_note").id,
         )
 
+    @api.model
```

Comment:
This shouldn't be `api.model`, as you may want to use `self` for selecting the partner.

Reply:
You're right. Fixed